homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -49,24 +51,32 @@ rules:
   entity-translations: done
   entity-device-class: done
   devices: done
-  entity-category: todo
-  entity-disabled-by-default: todo
+  entity-category:
+    status: done
+    comment: only data entities
+  entity-disabled-by-default:
+    status: done
+    comment: only two data entities
   discovery:
     status: exempt
     comment: api only, nothing on local network to discover services
   stale-devices:
     status: exempt
-    comment: one device only
+    comment: one permanent device only
   diagnostics: todo
   exception-translations: todo
-  icon-translations: todo
-  reconfiguration-flow: todo
+  icon-translations:
+    status: exempt
+    comment: no custom icons
+  reconfiguration-flow:
+    status: exempt
+    comment: no configuration option
   dynamic-devices:
     status: exempt
-    comment: one device only
+    comment: one fixed device only
```

Comment:
why exactly is this not discoverable?

Reply:
Because until your comment I considered it as completely unchangeable and fixed since I first took over the integration the counter which is the base of the integration is defined in the initial config flow.

But now that I think of it the counter is auto discoverable and is a device.
The only thing I don't know is if it is possible to have multiple counter and if so how will it be defined on the website because I don't have this information.

To conclude I will need to do a complete refactor of the integration to do so.
Find a new unique id for the integration.
Counter no more defined in config flow but as a device...

So lots of work to do 🤣